packages/relayer/README.md

```diff
@@ -0,0 +1,11 @@
+# Bridge
+
+Functionality to listen for events and relay them across chains
+
+### Redis
+
```

Comment:
Please add instructions on Redis version to use, as well as configuration like the port here. 

A longer description of the Relayer service is missing here, all instructions on how to run the service should be able to be found in this file.